src/cli/commands/gsn-status.ts

```diff
@@ -27,8 +28,10 @@ const commander = gsnCommander(['n', 'h'])
   }
 
   const config = configureGSN({ relayHubAddress })
-  const contractInteractor = new ContractInteractor(new Web3.providers.HttpProvider(host), config)
-  const httpClient = new HttpClient(new HttpWrapper({ timeout: statusConfig.getAddressTimeout }), config)
+  const logger = createLogger(config.logLevel)
```

Comment:
a bit strange to read from a static objcet we just configured the level. make more sense to read from cmdline.

Reply:
Fixed